cmd/gateway/zcn/gateway-zcn.go

```diff
@@ -260,6 +260,11 @@ func (zob *zcnObjects) DeleteObjects(ctx context.Context, bucket string, objects
 	ops := make([]sdk.OperationRequest, 0, len(objects))
 	for _, object := range objects {
 		remotePath := filepath.Join(basePath, object.ObjectName)
+		_, err := getSingleRegularRef(zob.alloc, filepath.Clean(remotePath))
+		if err != nil {
```

Comment:
Why are we fetching the ref? If its not there it will be ignored by delete operation.

Reply:
@Hitenjain14  minio raised an error here when the requested file didnot exist. The deleteworker expected to work for that case you mentioned  might have ignored as minio raised issue stating invalid remote ref.

Reply:
Please check where is the error coming from, is it DoMultiOperation for delete if yes then we need to fix it there rather than adding a check here

Reply:
The above changes will impact the performance of delete, getting a singular file ref can take upto 50ms and we do this for every object.

Reply:
ok